nwb_conversion_tools/gui/nwb_conversion_gui.py

```diff
@@ -679,11 +696,16 @@ def wheelEvent(self, event):
 
 
 # If it is imported as a module
-def nwb_conversion_gui(metafile=None, conversion_module='', source_paths={},
-                       kwargs_fields={}, extension_modules={}, extension_forms={},
-                       show_add_del=False):
+def nwb_conversion_gui(metafile=None, conversion_module=None, source_paths=None,
+                       kwargs_fields=None, extension_modules=None, extension_forms=None,
+                       show_add_del=False, nwbfile_loc=None, conversion_class=None):
     """Sets up QT application."""
+    if conversion_module:
+        raise DeprecationWarning('use of conversion_module will be replaced by conversion_class'
+                                 'with its \'run_conversion\' method called directly')
     app = QtCore.QCoreApplication.instance()
+    if conversion_class is None and conversion_module is None:
+        raise Exception('provide one of conversion_module:str or conversion_class:class')
     if app is None:
```

Comment:
If both are conversion_module and conversion_class are provided, it should result in an error